<launch>
    <arg name="robot_config" default="$(find gello_ros)/config/ur5e.yaml" />
    <arg name="common_config" default="$(find gello_ros)/config/common.yaml" />
    <arg name="save_episode" default="false" />
    <arg name="node_start_delay" default="10.0" />
    <arg name="agent_name" default="gello" />

    <include file="$(find gello_ros)/launch/realsense.launch" />
    <node name="gello_camera_server" pkg="gello_ros" type="run_realsense_ros_nodes.py"
        output="screen" launch-prefix="bash -c 'sleep $(arg node_start_delay); $0 $@' ">
        <rosparam command="load" file="$(arg robot_config)" />
        <rosparam command="load" file="$(arg common_config)" />
    </node>
</launch>